src/libraries/Microsoft.Extensions.Hosting/tests/UnitTests/ConsoleLifetimeExitTests.cs

```diff
@@ -105,10 +105,10 @@ await Host.CreateDefaultBuilder()
                         services.AddHostedService<EnsureEnvironmentExitCodeWorker>();
                     })
                     .RunConsoleAsync();
-            });
+            }, new RemoteInvokeOptions() { ExpectedExitCode = 124 });
 
+            // TODO: Remove once https://github.com/dotnet/arcade/issues/5865 is resolved
```

Comment:
It might work better to update to the new version and fix the tests in the same PR.

Then the test can follow the normal form for RemoteExecutor:
```C#
RemoteExecutor.Invoke(async () =>
{
}, new RemoteInvokeOptions() { ExpectedExitCode = 124 }).Dispose();
```

Reply:
I was looking to get these clean against both current and new arcade and then clean up the tests after the new behaviour flows over.